src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandlerTest.java

```diff
@@ -36,14 +36,14 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "indentation" + File.separator + filename);
+                + "indentation" + File.separator + "newhandler" + File.separator + filename);
     }
 
     @Test
     public void testInvalidLabel() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(IndentationCheck.class);
```

Comment:
@romani Should this be attached to this issue or should it be a minor?

@subkrish Please prepare the changes we are suggesting.
This test and it's inputs should move to `IndentationCheckTest` and this whole `NewHandlerTest` should be deleted.